master/getting-started/kubernetes/index.md

```diff
@@ -19,17 +19,21 @@ To deploy a cluster suitable for production, refer to [Installation](/{{page.ver
 - 2CPU
 - 2GB RAM
 - 10GB free disk space
-- RedHat Enterprise Linux 7.x+, CentOS 7.x+, Ubuntu 16.04+, or Debian 8.x+
+- RedHat Enterprise Linux 7.x+, CentOS 7.x+, Ubuntu 16.04+, or Debian 9.x+
```

Comment:
Why this change?

Reply:
It is as per kubeadm https://kubernetes.io/docs/setup/independent/install-kubeadm/